docs/proposals/instrumentation-of-services.md

```diff
@@ -0,0 +1,70 @@
+# Best practices for instrumenting infrastructure components in OpenShift
+
+All OpenShift components MUST expose the following endpoints on either their public HTTPS port, or a public HTTP port if none exists
```

Comment:
For components we control ourselves, using the HTTPS/HTTP port with these default paths probably makes sense.

If we are dealing with other components, I think we need to make this configurable via annotations. This could be used to point to a different port to be used, or also potentially point to a different path if for instance metrics are being exposed already but under a different location